data/cities/en/zurich.md

```diff
@@ -1,6 +1,9 @@
 ---
 slug: zurich
 title: Zürich
+description: |
```

Comment:
can we call those `socialDescription`? so is clear their purpose.

Reply:
It's being used in `<meta name="description">` and the open graph description and the twitter description. It's not necessary social.

Reply:
then probably `pageDescription`.
I'm just afraid it can be confusing for someone (i.e. other chapters) that just want to update their page. 
They don't have clue how the website works and they can get confuse which part they have to modify for which part of the website.
Is just easier if is super clear, without possibility to get confused

Reply:
I don't see how this will make a big difference but sure can do it. Maybe a comment would help more?

Reply:
We have `siteDescription` all over the code base, which actually should rather be `pageDescription`.